pkg/cmd/tests/tests_run.go

```diff
@@ -113,18 +89,37 @@ func NewRunOptions(streams genericclioptions.IOStreams) *RunOptions {
 		ParallelShard:         0,
 		ParallelServerAddress: "",
 		ParallelLogLevel:      0,
+
+		testSuites: testSuites,
 	}
 }
 
-func NewRunCommand(streams genericclioptions.IOStreams) *cobra.Command {
-	o := NewRunOptions(streams)
+func NewRunCommand(streams genericclioptions.IOStreams, customTestSuites ginkgotest.TestSuites, customOptions ...CustomFrameworkOptions) *cobra.Command {
+	testSuites := make(ginkgotest.TestSuites, 0, len(customTestSuites)+1)
+	for _, cts := range customTestSuites {
+		testSuites = append(testSuites, cts)
+	}
+
+	testSuites = append(testSuites, &ginkgotest.TestSuite{
+		Name: "all",
+		Description: templates.LongDesc(`
+		Runs all tests.`,
+		),
+		DefaultParallelism: 42,
+	})
```

Comment:
this should be part of the default suite as it used to

Reply:
done